web/shapeworks/src/components/AnalysisTab.vue

```diff
@@ -98,9 +274,29 @@ export default defineComponent({
         watch(stdDev, updateFileShown)
         watch(props, updateFileShown)
         watch(analysis, updateFileShown)
+        watch(groupSet, () => {
+            setDefaultPairing(); 
+            updateGroupFileShown();
+        })
+        watch(currPairing.value, updateGroupSelections)
```

Comment:
I don't think you need `.value` here, given the way that `watch` works

Reply:
The `.value` here is because without it, the `left` and `right` values aren't watched.